src/plugins/share/public/types.ts

```diff
@@ -51,6 +52,8 @@ export interface ShareContext {
   onClose: () => void;
   showPublicUrlSwitch?: (anonymousUserCapabilities: Capabilities) => boolean;
   disabledShareUrl?: boolean;
+  toasts: ToastsSetup;
+  intl?: InjectedIntl;
```

Comment:
Would it be possible to not make this optional? I'm guessing it would cause issues in Jest tests, which can be remedied.

```suggestion
  intl: InjectedIntl;
```